src/providers/MdnProvider.js

```diff
@@ -79,6 +79,12 @@ export function mdnSupported(node: Node, { version, target }: Target): boolean {
   // If a version is true then it is supported but version is unsure
   if (typeof versionAdded === 'boolean') return versionAdded;
   if (versionAdded === null) return true;
+
+  // Special case for Safari TP: TP is always gte than any other releases
```

Comment:
let's also handle when `customCoerce(version)` is `null`, so the general problem is solved.

Reply:
I thought about doing that but is it safe to assume all cases when customCoerce(version) is null should be treated like `Safari TP`?

Reply:
no, but i don't want non-safari null cases to crash in the future.

Reply:
Makes sense, updated.